Lib/test/test_minidom.py

```diff
@@ -1560,5 +1561,21 @@ def testProcessingInstructionNameError(self):
         pi = doc.createProcessingInstruction("y", "z")
         pi.nodeValue = "crash"
 
+    def test_minidom_attribute_order(self):
+        doc = Document()
+        root = doc.createElement("curriculum")
+        root.setAttribute( "status", 'public' )
+        root.setAttribute( "company", 'example' )
+        doc.appendChild(root)
+
+        with open(support.TESTFN, "w") as xml_file:
+            doc.writexml(xml_file)
```

Comment:
I am surprised, but `writexml()` was not tested before. `toxml()` and `toprettyxml()` are tested. Please add tests also for these methods.

Reply:
Done